pkg/api/errors.go

```diff
@@ -11,3 +21,69 @@ type AuthError struct {
 type InternalError struct {
 	error
 }
+
+type ErrorHandler struct {
+	logger *zap.Logger
+}
+
+func NewErrorHandler(logger *zap.Logger) ErrorHandler {
+	return ErrorHandler{
+		logger: logger,
+	}
+}
+
+func (eh *ErrorHandler) Handle(w http.ResponseWriter, r *http.Request, err error) {
+	if err == nil {
+		return
+	}
+	switch innerErr := errors.Cause(err).(type) {
+	case BadRequestError, *BadRequestError:
+		// TODO(nickeskov): remove this
+		http.Error(w, fmt.Sprintf("Failed to complete request: %s", innerErr.Error()), http.StatusForbidden)
+	case AuthError, *AuthError:
+		// TODO(nickeskov): remove this
```

Comment:
`// TODO(nickeskov): remove this` when exactly are you going to remove `this`? delete comments here or remove something that you was going to